server/routes/home/routes.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express'
+import HomeController from './controller'
+import logPageViewMiddleware from '../../middleware/logPageViewMiddleware'
+import AuditService from '../../services/auditService'
+
+const HomeRoutes = (auditService: AuditService): Router => {
+  const router = Router({ mergeParams: true })
+  const controller = new HomeController()
+
+  router.get('/', logPageViewMiddleware(auditService, controller), controller.GET)
+  return router
```

Comment:
Do you think the file name `routes.ts` could reflect the routes name i.e. `homeRoutes.ts` containing `HomeRoutes`?